ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2527,17 +2469,6 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
-- as: gcp-ipi-private-f6-disasterrecovery
```

Comment:
I suggest to untouch `disasterrecovery` jobs, because they are doing some adjustments recently. 

@geliu2016 this remind me, do you notice there are some existing disasterrecovery jobs in arm arch?
```
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: aws-ipi-private-f14-disasterrecovery
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: azure-ipi-private-f14-disasterrecovery
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: gcp-ipi-private-f14-disasterrecovery
```
Based on your last PR which change the disasterrecovery frequency, I guess you did not take into arm arch jobs into consideration.

Reply:
I will recover disasterrecovery jobs

Reply:
fixed